src/backends/wasm_js.rs

```diff
@@ -46,14 +66,16 @@ pub fn fill_inner(dest: &mut [MaybeUninit<u8>]) -> Result<(), Error> {
 
 #[wasm_bindgen]
 extern "C" {
-    // Return type of js_sys::global()
-    type Global;
     // Web Crypto API: Crypto interface (https://www.w3.org/TR/WebCryptoAPI/)
     type Crypto;
-    // Getters for the Crypto API
-    #[wasm_bindgen(method, getter)]
-    fn crypto(this: &Global) -> Crypto;
+    // Holds the global `Crypto` object.
+    #[wasm_bindgen(thread_local_v2, js_name = crypto)]
```

Comment:
I initially avoided using `globalThis` here because I have no idea how it works exactly on Node.js. In the meantime I have looked up the documentation and asked around: it should be perfectly fine to use `globalThis` in Node.js.

For context: the current setup just accesses the object directly from the global scope. According to the spec this is perfectly fine. However, this is usually discouraged because global variables can simply be overwritten. While I'm not sure if there is more history behind this, fact is that read-only properties like `crypto` can't be overwritten. So both approaches, accessing `Crypto` through the global `crypto` variable or `globalThis.crypto`, will work the same.

But to stick with "how it should be done" I will add `globalThis` now.